src/ziggurat/config.clj

```diff
@@ -61,8 +61,17 @@
       (slurp)
       (edn/read-string)))
 
+(defn- disable-exponential-backoff [config-map]
```

Comment:
If we implement it this way, I want a generic way to disable or enable experimental features.
Also why did you not use (ziggurat.util.map/nested-map-keys)?